lms/static/sass/application.scss

```diff
@@ -1,22 +0,0 @@
-/*
```

Comment:
@singingwolfboy, @mattdrayer: It seems like this file (`application.scss`) has been replaced by `lms-main.scss` and is no longer necessary. Likewise for its `-rtl` counterpart, and `course.scss` (replaced by `lms-course.scss`) Is that correct?